<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Histogram</class>
 <widget class="QDialog" name="Dialog_Histogram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>653</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>201</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Max_Label">
      <property name="text">
       <string>Max Value = </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MaxPercent_Label">
      <property name="text">
       <string>Max percent = </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MaxPos_Label">
      <property name="text">
       <string>Max Position = </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
